recipes/argopt/LICENSE

```diff
@@ -0,0 +1,10 @@
+* files: *
```

Comment:
The license file is included in the source distribution, so you can remove the manually added license file here.

Reply:
Removed in new commit.